cfme/infrastructure/host.py

```diff
@@ -226,12 +226,12 @@ def execute_button(self, button_group, button, cancel=True):
 
     def power_on(self):
         navigate_to(self, 'Details')
-        pow_btn('Power On')
+        pow_btn('Power On', invokes_alert=True)
```

Comment:
So doe we need to "version pick" behaviour dependant on 5.6 and 5.7 because the alert won't be present in 5.6?